.github/workflows/basic.yml

```diff
@@ -81,7 +81,7 @@ jobs:
       - name: Install apt packages
         run: |
           sudo apt-get update
-          sudo apt-get install -y clang cmake libnuma-dev libjemalloc-dev libtbb-dev
+          sudo apt-get install -y clang cmake libhwloc-dev libnuma-dev libjemalloc-dev libtbb-dev
```

Comment:
`linuma-dev` can be removed now?

Reply:
It's still needed in tests. We could use hwloc in tests as well but I think using numa gives us more confidence that the integration is done correctly.